aca-docker/mocks/wiremock/__files/prd_users_Org-Two.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please delete this file as well as exclude it from git via `.gitignore`

Reply:
I excluded it via `.gitignore`... but I did it after commiting the file 🤦 

File deleted